c_binding/include/libstoragemgmt/libstoragemgmt_local_disk.h

```diff
@@ -99,6 +99,36 @@ int LSM_DLL_EXPORT lsm_local_disk_vpd83_get(const char *disk_path,
                                             char **vpd83,
                                             lsm_error **lsm_err);
 
+/** New in version 1.4.
+ * Query the SMART Status of a given disk path.
+ * @param[in]   disk_path
+ *                  String. The disk path, example "/dev/sdc".
+ * @param[out]  smart_status
+ *                  Output pointer of int32_t.
+ *                          -1 (LSM_DISK_SMART_STATUS_UNKNOWN):
+ *                              SMART Status failure.
+ *                           0 (LSM_DISK_SMART_STATUS_FAIL):
+ *                              SMART Status failure.
+ *                           1 (LSM_DISK_SMART_STATUS_WARN):
+ *                              SMART Status warning.
+ *                           2 (LSM_DISK_SMART_STATUS_GOOD):
+ *                              SMART Status good.
+ * @param[out] lsm_err
+ *                  Output pointer of lsm_error. Error message could be
+ *                  retrieved via lsm_error_message_get(). Memory should be
+ *                   by lsm_error_free().
+ * @return Error code as enumerated by \ref lsm_error_number.
+ * @retval LSM_ERR_OK               on success or not found.
```

Comment:
I'm trying to wrap my head around the `or not found`.  I see this is also present on some of the other calls, but at the moment it's not making sense to me?